docs/guides/connecting_to_your_data/database/postgres.md

```diff
@@ -0,0 +1,146 @@
+---
+title: How to connect to your data in a postgresql database
+---
+import AddingCredentials from '../reference/database_credentials.md'
+import NextSteps from '../reference/link-to-validator.md'
+import ConnectionStringDetails from '../reference/configuration_explanation.md'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide will help you connect Great Expectations to data in a Postgresql database. We will be building a `Datasource` configuration, and using it to verify connectivity. By the end of this how-to guide, you should have a `Batch` representing a slice of data in your database, and a working code snippet that can be used as a starting point for more complex configurations.
+
+:::note Prerequisites: This how-to-guide assumes you have already:
+- Completed the [Getting Started Tutorial.](../../tutorials/quick-start.md)
+- Have a working installation of Great Expectations.
+- Have data on Postgresql database that you would like to connect to.
+:::
+
+## Steps
+
+### 1. Determine how to add credentials to configuration
+
+Great Expectations provides multiple methods of providing credentials for accessing databases. For our current example will use a `connection_string`, but other options include providing an Environment Variable, and loading from a Cloud Secret Store.  
+
+For more information, please refer to [Additional Notes](#additional-notes).
+
+```
+postgresql+psycopg2://postgres:@localhost/test_ci
+```
+
+<details><summary><b>What's in the config?</b></summary>
+<p>
+<ConnectionStringDetails />
+</p>
+</details>
+
+### 2. Install required packages
+
+Next make sure you have the necessary packages for Great Expectations to connect to your postgres database.
+
+```console
+pip install sqlalchemy psycopg2
+```
```

Comment:
To make all the articles more consistent it might be better to handle package installation in the prerequisites section.

Reply:
For this, the existing pattern is that we have installation of required module be one of the first steps of the actual `how-to-guide` [Example](https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_datasources/how_to_configure_a_pandas_s3_datasource.html). I do see your point though. What if we moved it to be the first step?

Reply:
I didn't realize we had a precedent for it - thank you for pointing that out. 🙇 Depending on if/how we do the interactive thing we might need to adjust our approach, but for now this is great!